src/seedu/addressbook/storage/AddressBookDecoder.java

```diff
@@ -11,12 +11,7 @@
 
 import seedu.addressbook.data.AddressBook;
 import seedu.addressbook.data.exception.IllegalValueException;
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
-import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.*;
```

Comment:
Don't use wildcard imports. It is a coding standard violation. You may want to look into your IntelliJ configuration to see if it is doing so for you automatically (and turn it off if it is).